src/aggregator/aggregation/gauge.go

```diff
@@ -31,22 +31,24 @@ import (
 type Gauge struct {
 	Options
 
-	lastAt     time.Time
-	annotation []byte
-	sum        float64
-	sumSq      float64
-	count      int64
-	max        float64
-	min        float64
-	last       float64
+	resendMinMax bool
+	lastAt       time.Time
+	annotation   []byte
+	sum          float64
+	sumSq        float64
+	count        int64
+	max          float64
+	min          float64
+	last         float64
 }
 
 // NewGauge creates a new gauge.
-func NewGauge(opts Options) Gauge {
+func NewGauge(resendMinMax bool, opts Options) Gauge {
```

Comment:
why do we need an explicit feature flag for this? should be able to use the `resendEnabled` on the rule

Reply:
Hm yeah, that does make sense; I guess you wouldn't be blindly opting in without checking it was a valid rule to resend maxes on beforehand